data-raw/estimate-truncation.R

```diff
@@ -0,0 +1,29 @@
+library("EpiNow2")
+library("here")
+
+# get example case counts
+reported_cases <- example_confirmed[1:60]
+
+# define example truncation distribution (note not integer adjusted)
+trunc_dist <- dist_spec(
+  mean = convert_to_logmean(3, 2),
+  mean_sd = 0.1,
+  sd = convert_to_logsd(3, 2),
+  sd_sd = 0.1,
+  max = 10
+)
+
+# Use the make_truncated_data function to generate example data for
+# an example using estimate_truncation()
+example_truncation_data <- purrr::map(
+  c(20, 15, 10, 0),
+  make_truncated_data,
```

Comment:
do we want this an internal function or do we want to define it here?

Reply:
I was debating where was best to store the function and agree it should live here as the whole point of this script is to make the data generation process standalone.